otx/algorithms/detection/init_venv.sh

```diff
@@ -108,6 +108,10 @@ echo torch=="${TORCH_VERSION}" >> "${CONSTRAINTS_FILE}"
 echo torchvision=="${TORCHVISION_VERSION}" >> "${CONSTRAINTS_FILE}"
 pip install torch=="${TORCH_VERSION}" torchvision=="${TORCHVISION_VERSION}" -f https://download.pytorch.org/whl/lts/1.8/torch_lts.html || exit 1
 
+# Install mmcv
+pip install --no-cache-dir mmcv-full==${MMCV_VERSION} || exit 1
+sed -i "s/force=False/force=True/g" "${venv_dir}"/lib/python"${PYTHON_VERSION}"/site-packages/mmcv/utils/registry.py  # Patch: remedy for MMCV registry collision from mmdet/mmseg
+
```

Comment:
Is this still needed?

Reply:
I just wanted to align other scripts, for example, [segmentation](https://github.com/cih9088/training_extensions/blob/feature/otx-new-mmcv/mmseg/otx/algorithms/segmentation/init_venv.sh#L111-L113), [classification](https://github.com/cih9088/training_extensions/blob/feature/otx-new-mmcv/mmseg/otx/algorithms/classification/init_venv.sh#L111-L113).
If this is not used, we should remove it from others as well.

Reply:
Let's remove it in other PR.